задание 5.2 вариант 9/Program.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace задание_5._2_вариант_9
+{
+    class Program
+    {
+        static void Main()
+        {
+            Console.WriteLine("Получить из слова кратер слова катер и река");
+
+            string str = "";
+
+            str = "Кратер";//к-0 р-1 а-2 т-3 е-4 р-5
+            Console.WriteLine(str);
+
+            string result1 = str.Substring(0, 1)
+                + str.Substring(2);
+            Console.WriteLine(result1);
+
+            string result2 = str.Substring(5).ToUpper()
+                + str.Substring(4, 1)
+                + str.Substring(0, 1).ToLower()
```

Comment:
По заданию исходное слово "кратер" и так должно быть с маленькой буквы. Использование метода `ToLower()` не требуется.
```suggestion
                + str.Substring(0, 1)
```